src/Microsoft.Health.Fhir.Shared.Api/Features/Formatters/FhirJsonInputFormatter.cs

```diff
@@ -71,7 +70,7 @@ public override Task<InputFormatterResult> ReadRequestBodyAsync(InputFormatterCo
 
                 try
                 {
-                    model = _parser.Parse<Resource>(jsonReader);
+                    model = await _parser.ParseAsync<Resource>(jsonReader);
```

Comment:
Do we still need `context.HttpContext.AllowSynchronousIO();` now?

Reply:
Removed